src/hotspot/share/opto/superword.hpp

```diff
@@ -219,9 +485,8 @@ class SuperWord : public ResourceObj {
 
   bool     do_vector_loop()        { return _do_vector_loop; }
 
-  const GrowableArray<Node_List*>& packset() const { return _packset; }
+  const PackSet& packset()   const { return _packset; }
```

Comment:
Somehow a strange alignment. You might want to fix that.

Reply:
Had it aligned with `do_vector_loop`. But it seems unnecessary, changed it.